src/sentry/api/endpoints/organization_tags.py

```diff
@@ -35,6 +43,7 @@ def get(self, request: Request, organization) -> Response:
                     filter_params["start"],
                     filter_params["end"],
                     use_cache=request.GET.get("use_cache", "0") == "1",
+                    dataset=Dataset(request.GET.get("dataset", "events")),
                     # Defaults to True, because the frontend caches these tags globally
                     include_transactions=request.GET.get("include_transactions", "1") == "1",
```

Comment:
If this is the only caller of this function, you may be able to replace `include_transactions` with `dataset` actually. But you will still want the default behavior to be the same (dataset: Discover)

Reply:
It looks like this is indeed the only caller of this function, so i'll implement this, thanks!